deployment/entity-service/templates/configmap.yaml

```diff
@@ -59,9 +59,7 @@ data:
   CHUNK_SIZE_AIM: {{ required "workers.CHUNK_SIZE_AIM is required." .Values.workers.CHUNK_SIZE_AIM | quote }}
 
   MAX_CACHE_SIZE: {{ required "workers.MAX_CACHE_SIZE is required." .Values.workers.MAX_CACHE_SIZE | quote }}
-
-  # ENTITY_MATCH_THRESHOLD not provided
```

Comment:
Why did you delete this two lines? The two variables are still in use in the file `settings.py`.

Reply:
This setting hasn't had any effect for a long time as the `threshold` is required when posting a new run - see spec [here](https://anonlink-entity-service.readthedocs.io/en/latest/api.html#operation/entityservice.views.run.list.post) - I'll create an issue (https://github.com/data61/anonlink-entity-service/issues/443) and aim to  remove it from settings too.